docs/reference/cluster/allocation-explain.asciidoc

```diff
@@ -25,6 +25,8 @@ GET _cluster/allocation/explain
 
 `GET _cluster/allocation/explain`
 
+`POST _cluster/allocation/explain`
```

Comment:
Good catch.  I assume that POST should have been documented already but we had missed it, but please correct me if that is not the case.  If that is the case, should we also be including examples for POST, similar to the ones we have for GET, or a note that they are equivalent?

Reply:
The convention is that all `GET` APIs that take a body should also work with `POST`, see e.g. https://www.elastic.co/guide/en/elasticsearch/reference/current/api-conventions.html#api-conventions

I've added this here to emphasise that you can send a request with a body - some tooling makes it hard to execute a `GET` with a body, you have to switch it into `POST` model

Reply:
Got it, @DaveCTurner thanks for clarifying.  Good to know there is general guidance in the documentation already.